src/App.spec.ts

```diff
@@ -641,7 +673,7 @@ describe('App', () => {
             // IncomingEventType.Command
             {
               body: {
-                command: '/COMMAND_NAME',
+                command: { command: '/COMMAND_NAME' },
```

Comment:
I don't think this is correct. The `body` should be like this:

```json
  "body": {
    "token": "legacy-verification-token",
    "team_id": "T123",
    "team_domain": "your-domain",
    "channel_id": "C123",
    "channel_name": "random",
    "user_id": "U123",
    "user_name": "seratch",
    "command": "/open-modal",
    "text": "",
    "response_url": "https://hooks.slack.com/commands/T123/123/random",
    "trigger_id": "123.123.random"
  },
```

Reply:
Nice catch, this does look weird, I think there is something else going on. 

I had to modify this for other reasons that I'm not remembering right now. 

I'll bet there's something going on.. Soon I'll make a PR to use a fixture lib I'm making b/c these are all a little wonky. 

https://github.com/IBM/slack-wrench/tree/master/packages/fixtures <- Shameless plug and present to anyone at Slack that wants to maintain it.